### DISCUSSION TITLE
MusicXML => MuseScore converter

### DISCUSSION THREAD

**Discussion by infojunkie** [w3c-cg/musicxml#564] [Show and tell]
Given the renewed importance of MuseScore now that Finale has been discontinued, and given the ongoing issues with MuseScore's own MusicXML import, I've started a dedicated MusicXML => MuseScore converter project. I hope that a focused effort on conversion can provide a more efficient development platform, as opposed to the massive codebase and process that MuseScore is burdened with. The converter is written in XSLT which I've found particularly well suited for XML transformations. 

Find the ongoing work here: https://github.com/infojunkie/musicxml-mscx - all contributions welcome!

**Comment by mscuthbert** [comment]
Thanks for the update!
